android/tangram/jni/platform_android.cpp

```diff
@@ -126,6 +180,29 @@ std::string stringFromResource(const char* _path) {
 
 }
 
+unsigned char* bytesFromExtMemory(const char* _path, unsigned int* _size) {
```

Comment:
This name is kinda confusing - it's reading from storage (not memory) and "Ext" is kinda ambiguous. Maybe `bytesFromFileSystem` ? `bytesFromStorage` ?